pkg/controller/controller_instance.go

```diff
@@ -302,6 +333,33 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) error {
 	glog.V(4).Infof("Provisioning a new Instance %v/%v of ServiceClass %v at Broker %v", instance.Namespace, instance.Name, serviceClass.Name, brokerName)
 	response, err := brokerClient.ProvisionInstance(request)
 	if err != nil {
+		// There are two buckets of errors to handle:
+		// 1.  Errors that represent a failure response from the broker
+		// 2.  All other errors
+		if osb.IsHTTPError(err) {
```

Comment:
Okay, we're working on that one and I'll update it in a separate PR and rebase onto it.